wtpython/display.py

```diff
@@ -17,11 +18,11 @@
 from wtpython.backends.stackoverflow import StackOverflowQuestion
 from wtpython.settings import APP_NAME
 
-PARSED_TB: dict[str, Any] = {}
+RAISED_EXC: Exception = None
```

Comment:
I know the storing in module stuff was there earlier as well, but this seems really hacky storing stuff in globals inside the `display.py` file. If we're doing it this way, I think it would be nicer to have the globals in `core.py`, or better just a separate file which only stores this data. That way we don't have globals floating around in the files where we're actually doing the important things

Reply:
before i stored them in display, i tried storing them in setttings.py. I had issues where the values were'nt keeping, so I moved it to display since that is the only module that should use the values. Ideally they would be passed to `Display` directly.